### ISSUE TITLE
Install dependencies fails on Debian Testing

### ISSUE THREAD

**Issue by rlafuente** [shoebot/shoebot#337]
```
shoebot/install - [master] » ./install_dependencies.sh 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package python-gi-cairo
E: Unable to locate package python-gobject
```

**Comment by rlafuente** [comment]
new names are `python3-gi-cairo` and `python3-gi`

fixed in 1f99bac

**Comment by stuaxo** [comment]
@rlafuente do the unit tests still pass after changing that ?

There is a test that checks the names match travis.
